graspy/inference/latent_distribution_test.py

```diff
@@ -14,121 +14,158 @@
     dot product graphs have the same distributions of latent positions.
-    
+
     This test can operate on two graphs where there is no known matching between
-    the vertices of the two graphs, or even when the number of vertices is different. 
+    the vertices of the two graphs.
     Currently, testing is only supported for undirected graphs.
 
     Read more in the :ref:`tutorials <inference_tutorials>`
 
     Parameters
     ----------
+    test : str
+        Backend hypothesis test to use, one of ["cca", "dcorr", "hhg", "rv", "hsic", "mgc"].
+        These tests are typically used for independence testing, but here they are used 
+        for a two-sample hypothesis test on the latent positions of two graphs. See 
+        :class:`hyppo.ksample.KSample` for more information. 
+
+    metric : str or function, (default="euclidean")
+        Distance metric to use, either a callable or a valid string.
+        The callable should behave similarly to :func:`sklearn.metrics.pairwise_distances`,
+        if a string should be one of the keys in `sklearn.metrics.pairwise.PAIRED_DISTANCES`
+        or "gaussian" which will use a Gaussian kernel on Euclidean distances.
+
     n_components : int or None, optional (default=None)
         Number of embedding dimensions. If None, the optimal embedding
-        dimensions are found by the Zhu and Godsi algorithm.
+        dimensions are found by the Zhu and Godsi algorithm. 
+        See :func:`~graspy.embed.selectSVD` for more information.
 
     n_bootstraps : int (default=200)
-        Number of bootstrap iterations.
+        Number of bootstrap iterations for the backend hypothesis test. See
+        :class:`hyppo.ksample.KSample` for more information.
 
-    bandwidth : float, optional (default=0.5)
-        Bandwidth to use for gaussian kernel. If None,
-        the median heuristic will be used.
+    workers : int, optional (default=1)
+        Number of workers to use. If more than 1, parallelizes the code.
 
     Attributes
     ----------
     sample_T_statistic_ : float
-        The observed difference between the embedded latent positions of the two 
+        The observed difference between the embedded latent positions of the two
         input graphs.
 
-    p_ : float
+    p_value_ : float
         The overall p value from the test.
-    
-    null_distribution_ : ndarray, shape (n_bootstraps, )
+
+    null_distribution_ : None or ndarray, shape (n_bootstraps, )
         The distribution of T statistics generated under the null.
 
     References
     ----------
-    .. [1] Tang, M., Athreya, A., Sussman, D. L., Lyzinski, V., & Priebe, C. E. (2017). 
+    .. [1] Tang, M., Athreya, A., Sussman, D. L., Lyzinski, V., & Priebe, C. E. (2017).
         "A nonparametric two-sample hypothesis testing problem for random graphs."
         Bernoulli, 23(3), 1599-1630.
+
+    .. [2] Panda, S., Palaniappan, S., Xiong, J., Bridgeford, E., Mehta, R., Shen, C., & Vogelstein, J. (2019).
```

Comment:
maybe this is just me, but it seems a little weird to cite a paper associated with a package that's imported anyway. I think this should have (a) theory paper(s) describing the methods applied to graphs instead. for example, one or multiple magic papers where graphs are used. thoughts?
cc @bdpedigo

Reply:
I don't see a reason to remove it, might as well promote hyppo. if you have other papers with mgc/graphs you are welcome to add them as citations. eventually this should just cite the paper bijan had on this? But i don't think that's out yet as far as i know

Reply:
sure. bijan's paper is available, but only as a preprint (but so is hyppo).